pages/common/pulumi-install.md

```diff
@@ -0,0 +1,12 @@
+# pulumi install
+
+> Install packages and plugins required by your existing project.
```

Comment:
```suggestion
> Install packages and plugins for the current program or policy pack.
```  
What about using the same description as in the documentation?

Reply:
I reworded it to highlight that a project needs to exist, else the command fails. I guess, the word "current" kind of implies existence.